src/web/pages/reports/auditdeltadetailspage.js

```diff
@@ -0,0 +1,570 @@
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useEffect, useState} from 'react';
+
+import {useDispatch, useSelector, shallowEqual} from 'react-redux';
+
+import {useRouteMatch} from 'react-router-dom';
+
+import _ from 'gmp/locale';
+
+import logger from 'gmp/log';
+
+import Filter, {RESET_FILTER, RESULTS_FILTER_FILTER} from 'gmp/models/filter';
+import {isActive} from 'gmp/models/task';
+
+import {first} from 'gmp/utils/array';
+import {isDefined, hasValue} from 'gmp/utils/identity';
+
+import withDownload from 'web/components/form/withDownload';
+
+import Reload, {
+  NO_RELOAD,
+  USE_DEFAULT_RELOAD_INTERVAL_ACTIVE,
+} from 'web/components/loading/reload';
+
+import withDialogNotification from 'web/components/notification/withDialogNotifiaction'; // eslint-disable-line max-len
+
+import DownloadReportDialog from 'web/pages/reports/downloadreportdialog';
+
+import {
+  loadAllEntities as loadFilters,
+  selector as filterSelector,
+} from 'web/store/entities/filters';
+
+import {
+  loadAllEntities as loadReportFormats,
+  selector as reportFormatsSelector,
+} from 'web/store/entities/reportformats';
+
+import {loadDeltaAuditReport} from 'web/store/entities/report/actions';
+
+import {deltaAuditReportSelector} from 'web/store/entities/report/selectors';
+
+import {
+  loadReportComposerDefaults,
+  renewSessionTimeout,
+  saveReportComposerDefaults,
+} from 'web/store/usersettings/actions';
+
+import {loadUserSettingDefaults} from 'web/store/usersettings/defaults/actions';
+import {getUserSettingsDefaults} from 'web/store/usersettings/defaults/selectors';
+import {getUserSettingsDefaultFilter} from 'web/store/usersettings/defaultfilters/selectors';
+
+import {
+  getReportComposerDefaults,
+  getUsername,
+} from 'web/store/usersettings/selectors';
+
+import compose from 'web/utils/compose';
+import {generateFilename} from 'web/utils/render';
+import PropTypes from 'web/utils/proptypes';
+import useGmp from 'web/utils/useGmp';
+
+import TargetComponent from '../targets/component';
+
+import Page from './deltadetailscontent';
+import FilterDialog from './detailsfilterdialog';
+
+const log = logger.getLogger('web.pages.report.deltadetailspage');
+
+const DEFAULT_FILTER = Filter.fromString(
+  'levels=hmlg rows=100 min_qod=70 first=1 sort-reverse=compliant',
+);
+
+const REPORT_FORMATS_FILTER = Filter.fromString('active=1 and trust=1 rows=-1');
+
+const getTarget = (entity = {}) => {
+  const {report = {}} = entity;
+  const {task = {}} = report;
+  return task.target;
+};
+
+const getFilter = (entity = {}) => {
+  const {report = {}} = entity;
+  return report.filter;
+};
+
+const DeltaAuditReportDetails = props => {
+  const [activeTab, setActiveTab] = useState(0);
+  const [showFilterDialog, setShowFilterDialog] = useState(false);
+  const [showDownloadReportDialog, setShowDownloadReportDialog] =
+    useState(false);
+  const [reportFormatId, setReportFormatId] = useState(undefined);
```

Comment:
As above I am relatively sure that `useState(undefined)` is the same as `useState()`.